tools/@aws-cdk/ubergen/bin/ubergen.ts

```diff
@@ -254,8 +254,9 @@ async function combineRosettaFixtures(libraries: readonly LibraryReference[]) {
 
   for (const library of libraries) {
     const packageRosettaDir = path.join(library.root, 'rosetta');
+    const uberRosettaTargetDir = path.join(uberRosettaDir, library.shortName.replace(/-/g, '_'));
     if (await fs.pathExists(packageRosettaDir)) {
-      await fs.copy(packageRosettaDir, uberRosettaDir, {
+      await fs.copy(packageRosettaDir, uberRosettaTargetDir, {
```

Comment:
I think we probably need to make an exception for the `core` submodule, which gets moved to the ROOT of the monopackage, not placed in a submodule.

Reply:
Now the fixtures show up in the `rosetta` directory and the number of outstanding failures went from 1973 to 1722. Also validated that `stack.AddTransform` compiles along with a few other examples from the core module.